src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx

```diff
@@ -117,10 +117,20 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="#1" xml:space="preserve">
+  <data name="1" xml:space="preserve">
```

Comment:
No, I made a mistake. When used in a string resource the hash means look up in this package, but when it looks it up, it strips the hash.